shell/platform/windows/window_win32.h

```diff
@@ -256,6 +259,9 @@ class WindowWin32 {
 
   // Generates touch point IDs for touch events.
   SequentialIdGenerator touch_id_generator_;
+
+  const static int kVsyncTimer = 1;
```

Comment:
Is it possible that this value coincides with some timer that is implicitly set by others? Should we use a more arbitrary value?

Reply:
The only other timer being set is in the task runner, which uses a timer ID of 0. Any future timer which uses ID 1 without checking for previous uses will notice their problem during testing since this one fires every frame.